tests/unit/views/test_dashboard.py

```diff
@@ -1,15 +1,22 @@
 import time
 import unittest
+import sys
 
 from tests.unit import AsyncHTTPTestCase
 from tests.unit.utils import task_succeeded_events, task_failed_events
 from tests.unit.utils import HtmlTableParser
 
+if sys.version_info >= (2, 7):
+    from mock import patch, PropertyMock
+else:
+    from unittest.mock import patch, PropertyMock
+
+
 from celery.events import Event
 from celery.utils import uuid
 
 from flower.events import EventsState
-
+from flower.options import options
```

Comment:
[PEP-8](https://www.python.org/dev/peps/pep-0008/#blank-lines):
> Surround top-level function and class definitions with two blank lines.

Please keep an empty line here.

```suggestion
from flower.options import options

```